exchange/auction.go

```diff
@@ -60,20 +64,39 @@ func (a *auction) setRoundedPrices(priceGranularity openrtb_ext.PriceGranularity
 	a.roundedPrices = roundedPrices
 }
 
-func (a *auction) doCache(ctx context.Context, cache prebid_cache_client.Client, targData *targetData, bidRequest *openrtb.BidRequest, ttlBuffer int64, defaultTTLs *config.DefaultTTLs, bidCategory map[string]string) []error {
-	var bids, vast, includeBidderKeys, includeWinners bool = targData.includeCacheBids, targData.includeCacheVast, targData.includeBidderKeys, targData.includeWinners
-	if !((bids || vast) && (includeBidderKeys || includeWinners)) {
+func (a *auction) doCache(
+	ctx context.Context,
+	cache prebid_cache_client.Client,
+	targData *targetData,
+	bidRequest *openrtb.BidRequest,
+	ttlBuffer int64,
+	defaultTTLs *config.DefaultTTLs,
+	bidCategory map[string]string,
+) []error {
+
+	if (!targData.includeCacheBids && !targData.includeCacheVast) || (!targData.includeBidderKeys && !targData.includeWinners) {
 		return nil
 	}
+
 	var errs []error
-	expectNumBids := valOrZero(bids, len(a.roundedPrices))
-	expectNumVast := valOrZero(vast, len(a.roundedPrices))
-	bidIndices := make(map[int]*openrtb.Bid, expectNumBids)
-	vastIndices := make(map[int]*openrtb.Bid, expectNumVast)
-	toCache := make([]prebid_cache_client.Cacheable, 0, expectNumBids+expectNumVast)
-	expByImp := make(map[string]int64)
-	competitiveExclusion := false
+	var expectNumBids int
+	var expectNumVast int
+	var bidIndices map[int]*openrtb.Bid
+	var vastIndices map[int]*openrtb.Bid
+	var toCache []prebid_cache_client.Cacheable
+	var expByImp map[string]int64
 	var hbCacheID string
+	var competitiveExclusion bool
+	var bids_to_cache_arr []*pbsOrtbBid
```

Comment:
Variable name police comment:  bids_to_cache_arr should be bidsToCacheArr.

Reply:
Done